bogo/src/main.rs

```diff
@@ -1195,6 +1194,25 @@ fn flush(sess: &mut Connection, conn: &mut net::TcpStream) {
     conn.flush().unwrap();
 }
 
+fn orderly_close(conn: &mut net::TcpStream) {
```

Comment:
nit: this file has the same issues elsewhere but I think `orderly_close()` should be positioned after `after_read()`.